site/app/models/RainbowCustomization.php

```diff
@@ -122,7 +122,12 @@ public function buildCustomization() {
                     $this->bucket_counts[$bucket] = $json_bucket->count;
                 }
 
-                $this->bucket_remove_lowest[$bucket] = $json_bucket->remove_lowest;
+                if (property_exists($json_bucket, 'remove_lowest')) {
+                    $this->bucket_remove_lowest[$bucket] = $json_bucket->remove_lowest;
+                }
+                else {
+                    $this->bucket_remove_lowest[$bucket] = 0;
+                }
```

Comment:
PHP's null coalescing operator works on properties too, so this can be simplified as follows:
```suggestion
                $this->bucket_remove_lowest[$bucket] = $json_bucket->remove_lowest ?? 0;
```